Simplenote/src/main/java/com/automattic/simplenote/authentication/SignInFragment.kt

```diff
@@ -101,6 +104,17 @@ class SignInFragment: MagicLinkableFragment() {
                 "wpcc_button_press_signin_activity"
             )
         }
+        val manualLoginTextView = view.findViewById<TextView>(R.id.sign_in_login_manually)
+        val message = getString(R.string.signin_login_with_email_manually);
+        val span = StrUtils.generateClickableSpannableString(LOGIN_MANUALLY_SUBSTRING, message
```

Comment:
I don't think this will work for localized strings... can it instead work like this string does? Not sure how it works with the placeholders:

`<string name="simperium_footer_signup">By creating an account, you agree to our %1$s%2$s%3$sTerms and Conditions%4$s</string>`

And the code that formats it:

```
String.format(
                if (this.isLogin) this.resources.getString(R.string.simperium_footer_login) else this.resources.getString(
                    R.string.simperium_footer_signup
                ), "<span style=\"color:#", colorLink, "\">", "</span>"
            )
```

Something like that?